Formula/p/periphery.rb

```diff
@@ -0,0 +1,45 @@
+class Periphery < Formula
+  desc "Identify unused code in Swift projects"
+  homepage "https://github.com/peripheryapp/periphery"
+  url "https://github.com/peripheryapp/periphery/archive/refs/tags/2.21.2.tar.gz"
+  sha256 "97e399df17d1e681703c5c8852e50b40c056a98b41dc4c615011c048b8348dbb"
+  license "MIT"
+  head "https://github.com/peripheryapp/periphery.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on xcode: ["15.4", :build]
+  depends_on macos: :sonoma
```

Comment:
would swift 6 work for ventura?

Reply:
I bumped it to `:ventura`  as that's the minimum version supported by the [Package.swift](https://github.com/peripheryapp/periphery/blob/master/Package.swift#L177).

Reply:
bumping to `:ventura` actually wont work

```
==> SKIPPED periphery
Warning: A full installation of Xcode.app 15.4 is required to compile
this software. Installing just the Command Line Tools is not sufficient.

Xcode 15.4 cannot be installed on macOS 13. 
```

https://github.com/Homebrew/homebrew-core/actions/runs/12508037405/job/34895612061#step:3:40